### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by hannahbus** [geopandas/geopandas#2764] [General]
I greatly enjoy using geopandas. Having said this, I was wondering whether there are plans to make the documentation more detailed? For instance, from the documentation on `geopandas.GeoSeries.representative_point`, it is unclear how the representative point is computed. Also, for people new to the GIS community, it could be helpful to extend the discussion on `geopandas.GeoSeries.distance` and briefly sketch distance computations in projected coordinate systems as opposed to geographic coordinate systems and what is supported in geopandas.

**Comment by martinfleis** [comment]
Hi, the documentation improvements are always welcome and you are right about the cases you mention. Any pull request in this direction would be highly appreciated but there are no specific plans to have a focused attention on documentation right now. 

If you're looking for a specific answer - geopandas operations like `distance` work in a planar space only, so projected coordinates are required and you get a warning if you use geographic coords. Operations done based on the spherical geometry are on the roadmap and development currently happens over at https://github.com/benbovy/spherely.

And an explanation of representative point is here - https://gis.stackexchange.com/a/414263

**Reply by hannahbus** [reply]
Many thanks for the feedback. I will push a pull request then, happy to help.
